fairseq/data/audio/text_to_speech_dataset.py

```diff
@@ -196,6 +197,7 @@ def _from_list(
         bpe_tokenizer,
         n_frames_per_step,
         speaker_to_id,
+        multitask=None,
```

Comment:
Doesn't seem to be used anywhere else, what is this for?

Reply:
@aghatare This is necessary because TextToSpeechDataset inherits SpeechToTextDataset.